emmaa/statements.py

```diff
@@ -174,7 +174,7 @@ def check_stmt(stmt, conditions, evid_policy='any'):
             metadata = emmaa_anns.get('metadata')
             checks = []
             for key, value in conditions.items():
-                checks.append(metadata[key] == value)
+                checks.append(key in metadata and metadata[key] == value)
```

Comment:
Alternate would be to only append to the `checks` list if `key in metadata`, then potentially add additional logic `if len(checks) == 0` after this loop over `conditions.items()`